phpunit.xml.dist

```diff
@@ -30,6 +30,7 @@
     <groups>
         <exclude>
             <group>large</group>
+            <group>e2e</group>
```

Comment:
This change looks a bit confusing - I would prefer it being introduced in a dedicated PR where we can talk more of the different PHPUnit groups

Reply:
This is probably due to me turnign it off quickly for local dev to check a few things faster, but i forgot to remove that